notebook2/notebook/notebook.py

```diff
@@ -498,19 +546,7 @@ def login_auth0():
         if workshop_password != PASSWORD:
             return redirect(external_url_for(f"error?err=Unauthorized"))
 
-        session['user'] = {
-            'id': uuid.uuid4().hex,
-            'name': "Guest",
-            'email': "N/A",
-            'picture': "N/A",
-        }
-
-        if 'referrer' in session:
-            referrer = session['referrer']
-            del session['referrer']
-            return redirect(referrer)
-
-        return redirect(external_url_for(''))
+        session['workshop_user'] = True
```

Comment:
This simply reverts pre-IBG2019 behavior: the user needs to have both an auth0 account and a workshop password, if they are not whitelisted (spoke to @cseed about this last week, but if I misunderstood, happy to revert).